### DISCUSSION TITLE
Website down

### DISCUSSION THREAD

**Discussion by Kyngo** [tteck/Proxmox#3182] [General]
Hello, it looks like the website is down.

![Captura de pantalla_20240612_211709](https://github.com/tteck/Proxmox/assets/1144200/9b081d83-7ee5-4875-b927-65ce97f68412)

Is the URL gonna change? Did I miss anything? I can't find any info if that's the case.

**Comment by tteck** [comment]
DNS issues use https://tteck.github.io/Proxmox/

**Reply by Kyngo** [reply]
Why does it look **so different** than I can remember? 🤔

**Reply by K3v-68** [reply]
what you see now is the old site ;)

**Comment by chenks** [comment]
is the previous site going to come back?
it had a much better UI compared to the one that we are now being directed to.

**Reply by tteck** [reply]
I liked it also, but the Vercel charges were unacceptable, gh-pages is free.

**Reply by chenks** [reply]
take it you can't make the gh-pages use the same UI and styling as the other site?

**Reply by OddSquirrel** [reply]
I think the problem is, that Vercel includes a lot of things that make it easy to integrate stuff from GitHub. @tteck could always just use some cheaper random server elsewhere, but then he'd have to constantly support that and work on it on top of what he does over here.

**Reply by tteck** [reply]
Disappointingly no.

**Reply by tteck** [reply]
> could always just use some cheaper random server elsewhere

Not with a Next.js site. Vercel or Cloudflare was about it, unless I wanted to selfhost using [Coolify](https://coolify.io/)

**Reply by OddSquirrel** [reply]
@tteck What I meant was that you could simply grab webspace, slap Wordpress / Hugo / raw HTML / whatever on it and merrily tend to it day in, day out. Next.js isn't a hard requirement for a random website AFAIK. 😉

**Reply by danielsanfr** [reply]
This site is safe?
https://proxmox-helper-scripts.vercel.app/scripts

**Reply by tteck** [reply]
Be cautious of copycat or coat-tailing sites that exploit the project's popularity with potentially malicious intent. Please only trust information from https://helper-scripts.com/ or https://tteck.github.io/Proxmox/.
